charts/cert-manager-issuers/templates/NOTES.txt

```diff
@@ -0,0 +1,6 @@
+{{- if .Values.issuers }}
+- Deployed Issuers
```

Comment:
It would be nice if we could output a list of issuers and/or clusterIssuers that have been deployed by the chart, but I was not able to figure out quickly how the `cert-manager-issuers.metadata` is generating the name of e.g. the sole `selfSigned` clusterIssuer.

Reply:
I'm gonna hate myself for how the code in NOTES.txt looks once I implement this but I do see a need for it.

Given that we don't usually look at the NOTES output when we use Argo CD I am tending to want to implement this in a future update to the chart. I'm hoping that this future update will also take into account any real world issues we encounter when we start using this chart (so I don't need to refactor the notes stuff twice).

I'll think some more about this, if I figure out a somewhat DRY way to do this I'll update this PR.

Reply:
I refactored the helper and rewrote NOTES.txt